android/app/src/main/java/org/unicef/ecar/bebbo/ExactAlarmModule.java

```diff
@@ -0,0 +1,50 @@
+package org.unicef.ecar.bebbo;
+
+import android.app.AlarmManager;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.provider.Settings;
+import androidx.annotation.NonNull;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+
+public class ExactAlarmModule extends ReactContextBaseJavaModule {
+
+    private final ReactApplicationContext reactContext;
+
+    ExactAlarmModule(ReactApplicationContext context) {
+        super(context);
+        this.reactContext = context;
+    }
+
+    @NonNull
+    @Override
+    public String getName() {
+        return "ExactAlarmModule";
+    }
+
+    @ReactMethod
+    public void requestExactAlarmPermission(Promise promise) {
+        try {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU || Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                AlarmManager alarmManager = (AlarmManager) reactContext.getSystemService(Context.ALARM_SERVICE);
+                if (alarmManager.canScheduleExactAlarms()) {
```

Comment:
// Check if permission is already granted
if (alarmManager != null && alarmManager.canScheduleExactAlarms()) {

Reply:
changed as you suggested